extensions/amp-script/amp-script.md

```diff
@@ -138,6 +138,51 @@ Will be reflected on the page as a new child of the `amp-script` element:
 
 Under the hood, `amp-script` uses [@ampproject/worker-dom](https://github.com/ampproject/worker-dom/). For design details, see the ["Intent to Implement" issue](https://github.com/ampproject/amphtml/issues/13471).
 
+### Advanced usages
+
+#### Passing data between amp-script and the page
+
+`amp-script` supports getting and setting [`amp-state`](https://amp.dev/documentation/components/amp-bind/#initializing-state-with-amp-state) JSON via JavaScript.
+
+This enables advanced interactions between `amp-script` and other AMP elements on the page via `amp-bind` [bindings](https://amp.dev/documentation/components/amp-bind/#bindings).
```

Comment:
Does this enable `amp-script` to interact with elements outside of the `<amp-script>` tag?

Reply:
Indirectly yes, by modifying `amp-state` which can affect elements that have bindings. 

Is the current phrasing too vague?

Reply:
After this sentence I would add:
"These elements can be inside or outside the `amp-script` tag."

Reply:
There's an edge case here: AMP elements inside `amp-script` doesn't really work well yet.

Reply:
Is this something we're working on? If not, should we advise developers to place AMP elements outside of `amp-script`?

Reply:
Added the suggested sentence.